cli_reference/cli_by_example_content.adoc

```diff
@@ -517,6 +520,24 @@
   $ oc create route reencrypt --service=backend --dest-ca-cert=ca.crt
 ----
 ====
+endif::[]
+
+
+ifdef::openshift-online[]
+== Securely expose containers via a route
+
+====
+
+[options="nowrap"]
+----
+  // Create a secure edge-terminated route.
+  $ oc create route edge --service=frontend
+
+  // Create a secure passthrough-terminated route.
```

Comment:
Should be just "passthrough", not "passthrough-terminated"... although I notice the original example says "passthrough terminated"...